package.json

```
@@ -27,15 +27,15 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@egendata/messaging": "0.6.0",
-    "@egendata/react-native-jose": "0.4.0",
+    "@egendata/react-native-jose": "file:../react-native-jose/egendata-react-native-jose-0.4.0.tgz",
```

```suggestion
    "@egendata/react-native-jose": "0.5.0",
```

when https://github.com/egendata/react-native-jose/pull/3 is released